test/framework/src/main/java/org/elasticsearch/test/rest/yaml/ObjectPath.java

```diff
@@ -148,4 +149,14 @@ private Object evaluate(String key, Object object, Stash stash) throws IOExcepti
 
         return list.toArray(new String[list.size()]);
     }
+
+    public XContentBuilder toXContentBuilder(XContent xContent) throws IOException {
```

Comment:
oh right, leave it here then, it's fine. How about making ObjectPath implement ToXContentObject at this point?